src/memory/mod.rs

```diff
@@ -151,3 +151,17 @@ pub(crate) fn memset(slice: &mut [u8], value: u8) {
         ptr::write_bytes(p, value, slice.len());
     }
 }
+
+pub fn load_c_string<M: Memory>(memory: &mut M, addr: &M::REG) -> Result<Bytes, Error> {
```

Comment:
Can we rename this function as `load_c_string_byte_by_byte`? Ideally, we should use an optimized version which loads a bulk of bytes at once as the general version. But it's also a concern not to change existing behavior. So if we do want to provide such a function that reads byte by byte, I do prefer we say it explicitly in the name that this is an un-optimized version

Reply:
The function has been renamed.